public/index.html

```diff
@@ -1,5 +1,11 @@
 <!doctype html>
-<html manifest="%PUBLIC_URL%/ios.manifest" lang="en">
+<script type="text/javascript">
+if(navigator.userAgent.match(/(iPhone|iPod|iPad)/i)) {
+  document.write('<html manifest="%PUBLIC_URL%/ios.manifest" lang="en">');
+} else {
+  document.write('<html lang="en">');
+}
+</script>
```

Comment:
Removing the <html> tag here makes the whole document invalid. Also, html element can only contain a head and a body tag.

Can you try this?
```
<!doctype html>
<html lang="en">
<head>
  all the meta tags...
  <script>
    if (navigator.userAgent.match(/(iPhone|iPod|iPad)/i)) {
      // BTW, this test is weak, eg IE Mobile also mentions iPhone in its user agent string...
      const htmlEl = document.getElementsByTagName('html')[0];
      htmlEl.setAttribute('manifest', '%PUBLIC_URL%/ios.manifest');

      // The rest of your manifest update detection code goes here, inside the user agent test.
      // ...
    }
  </script>
</head>
<body>
...
```